scripts/start-local-app.js

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const childProcess = require('child_process');
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const openBrowser = require('react-dev-utils/openBrowser');
+
+const [, , url] = process.argv;
+
+const queryParams = childProcess.execFileSync('node', [
```

Comment:
Why do we need the child process?

Reply:
It is another node script that doesn't return a function or similar. It writes the result to stdout.